data/jobs.ts

```diff
@@ -2,6 +2,16 @@ import type { JobType } from "@/types/job"
 import { generateId } from "@/lib/utils"
 
 export const jobs: JobType[] = [
+  {
+  id: generateId(), // Leave this as is, it will generate a unique ID
+  title: "🛠️ Founding Engineer", 
+  description: "We are looking for a founding engineer to support our clients in the U.S. and E.U.",
+  company: "Qurio - Growing Newsrooms at the Speed of Trust",
+  companyDescription: "We are a team of journalists and engineers redefining how newsrooms can deliver greater relevance and value to readers, gaining greater value in return. ",
```

Comment:
can we make this shorter? "a team of journalists and engineers redefining how newsrooms can deliver greater relevance and value to readers"